tensorflow/python/ops/lookup_ops.py

```diff
@@ -1849,14 +1849,31 @@ def remove(self, keys, name=None):
 
     return op
 
-  def lookup(self, keys, name=None):
+  def lookup(self, keys, dynamic_default_values=None, name=None):
     """Looks up `keys` in a table, outputs the corresponding values.
 
     The `default_value` is used for keys not present in the table.
 
     Args:
       keys: Keys to look up. Can be a tensor of any shape. Must match the
         table's key_dtype.
+      dynamic_default_values: The values to use if a key is missing in the
+        table. If None (by default), the `table.default_value` will be used.
+        Shape of `dynamic_default_values` must be same with
+        `table.default_value` or the lookup result tensor.
```

Comment:
Hi @rohan100jain, I just change the comments about `table.default_value`, Please check it and maybe you have to approved it again , Thanks!
[...](https://github.com/tensorflow/tensorflow/pull/43269#discussion_r511370761)